mkdocs.yml

```diff
@@ -26,5 +26,6 @@ nav:
   - Home: 'index.md'
   - Pastas & similar:
       - Gnocchi with Pumpkin: "main-courses/pastas/gnocchi_with_pumpkin.md"
+      - Pastasallad: "main-cources/pastaspastasallad.md"
```

Comment:
Did you forget a / in this line? Looks like the recipe is not found:

![image](https://user-images.githubusercontent.com/28093618/140755689-8f003412-a82b-4faa-8281-58c623791d43.png)